pkg/envoy/rds/response.go

```diff
@@ -37,6 +37,16 @@ func NewResponse(catalog catalog.MeshCataloger, proxy *envoy.Proxy, _ *xds_disco
 		TypeUrl: string(envoy.TypeRDS),
 	}
 
+	allTrafficSplits, _, _, _, _ := catalog.ListSMIPolicies()
+	isTrafficSplitService := func(svc service.MeshService) bool {
+		for _, split := range allTrafficSplits {
+			if split.Namespace == svc.Namespace && split.Spec.Service == svc.Name {
+				return true
+			}
+		}
+		return false
+	}
```

Comment:
Could we carve this out as a separate function so that it is unit testable ? 

Once separated please add a unit test

Reply:
Yes, will provide ut laster.